src/buffer/out/textBuffer.cpp

```diff
@@ -1059,6 +1065,10 @@ std::string TextBuffer::GenHTML(const TextAndColor& rows, const int fontHeightPo
 {
     try
     {
+        // TODO: the font name needs to be passed and stored around as an actual bounded type, not an implicit bounds on LF_FACESIZE

```

Comment:
untagged todo

Reply:
This will/should be solved by #602 so I tagged that one.